build_defs/build_defs.bzl

```diff
@@ -17,14 +18,14 @@ def _run_schema2pkg(name, src, out, language_name, language_flag, file_extension
         name = name,
         srcs = [src],
         outs = [out],
-        deps = ["//src/tools:schema2pkg_srcs"],
+        deps = ["@arcs//src/tools:schema2pkg_srcs", "@arcs//tools"],
         # TODO: generated header guard should contain whole workspace-relative
         # path to file.
-        cmd = "./tools/sigh schema2pkg " +
+        cmd = "$(location @arcs//tools) schema2pkg " +
```

Comment:
This doesn't seem to work. When I try testing locally I get:

```
==================== Test output for //src/wasm/cpp/tests:wasm-cpp-test:
/usr/local/google/home/csilvestrini/.cache/bazel/_bazel_csilvestrini/3b285f724e0b71687487035dc2909458/execroot/arcs/bazel-out/k8-fastbuild/bin/src/wasm/cpp/tests/wasm-cpp-test.sh.runfiles/arcs/src/wasm/cpp/tests/wasm-cpp-test.sh: line 3:
./tools/sigh: No such file or directory
================================================================================
```

Reply:
It also seems to be failing on Travis (with a different error though):

```
ENOENT: no such file or directory, open '/home/travis/build/PolymerLabs/arcs/home/travis/.cache/bazel/_bazel_travis/504270edcd9a48d6d285f3ee471e3abc/execroot/arcs/src/wasm/cpp/tests/schemas.arcs'
Post-parse processing error caused by './in-memory-1773926.manifest' line 1.
Error importing '/home/travis/build/PolymerLabs/arcs//home/travis/.cache/bazel/_bazel_travis/504270edcd9a48d6d285f3ee471e3abc/execroot/arcs/src/wasm/cpp/tests/schemas.arcs'
  import '/home/travis/build/PolymerLabs/arcs//home/travis/.cache/bazel/_bazel_travis/504270edcd9a48d6d285f3ee471e3abc/execroot/arcs/src/wasm/cpp/tests/schemas.arcs'
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
No schemas found in '/home/travis/.cache/bazel/_bazel_travis/504270edcd9a48d6d285f3ee471e3abc/execroot/arcs/src/wasm/cpp/tests/schemas.arcs'
```

Reply:
Yeah, it's weird, works on Kotlin if you build from the src/javaharness directory, but not from outside. It appears related to nested workspaces, I'm trying to fix it now. I think if we get rid of the nested WORKSPACE, it would be simpler, but then non-nested WORKSPACE leads to to nastier paths everywhere.

Reply:
Shouldn't it be: `$(location @arcs//tools)/sigh`?
Does `@arcs/tools` point to sigh specifically or the tools directory?

Reply:
Just read the tools BUILD file. 

Does it make sense to have it be `${location @arcs//tools):sigh`?

Reply:
its fixed now, the only restriction is "local" execution due to Docker privileges for creating symlinks.